common/lib/xmodule/xmodule/modulestore/tests/utils.py

```diff
@@ -72,6 +73,28 @@ def mock_tab_from_json(tab_dict):
     return tab_dict
 
 
+def add_temp_files_from_dict(file_dict, dir):
+    """
+    Takes in a dict formatted as: { file_name: content }, and adds files to directory
+    """
+    for file_name in file_dict:
+        sample_file = open("{}/{}".format(dir, file_name), "w+")
```

Comment:
nit: I think we  prefer context managers for file IO. Not sure if it makes it easier for the python3 upgrade.